Dockerfile

```diff
@@ -9,12 +9,11 @@ COPY rpms /rpms/
 COPY dnf-docker-test/x509certgen /usr/local/bin
 
 RUN set -x && \
-    echo -e "deltarpm=0\ntsflags=nodocs" >> /etc/dnf/dnf.conf && \
+    echo -e "deltarpm=0" >> /etc/dnf/dnf.conf && \
```

Comment:
why do we want docs? ;)

Reply:
Now I remember why I did remove it. These tsflags settings is already present in dnf.conf and thus this echo was adding duplicate record which subsequently caused issues in dnf.conf parsing in the ini file step.